packages/framework/src/Component/Domain/Domain.php

```diff
@@ -207,4 +207,18 @@ public function getDateTimeZone(): DateTimeZone
     {
         return $this->getCurrentDomainConfig()->getDateTimeZone();
     }
+
+    /**
+     * @return array<int, string>
+     */
+    public function getAllDomainsAsChoices(): array
```

Comment:
When it's named `...asChoice`, shouldn't it be already in format required for choice? 
Now it's necessary to call array_flip